src/main/java/com/example/SimpleConnection.java

```diff
@@ -92,10 +78,31 @@ private void loadProjects()
     public static void main(String[] args) throws Exception {
         SimpleConnection client = new SimpleConnection();
         try {
-            client.connect(args);
-            //Do something e.g. loading user's data.
-            //Load the projects/datasets owned by the user currently logged in.
-            client.loadProjects();
+            String host = null;
+            int port = 4064;
+            String user = null;
+            String password = null;
+            for (String arg : args) {
+                String[] tmp = arg.split("=");
+                tmp[1].replace("'", "");
+                tmp[1].replace("\"", "");
+                if (tmp[0].trim().equals("--omero.host"))
+                    host = tmp[1].trim();
+                if (tmp[0].trim().equals("--omero.port"))
+                    port = Integer.parseInt(tmp[1].trim());
+                if (tmp[0].trim().equals("--omero.user"))
+                    user = tmp[1].trim();
+                if (tmp[0].trim().equals("--omero.pass"))
+                    password = tmp[1].trim();
+            }
+
+            if (host != null && user != null && password != null) {
+                client.connect(host, port, user, password);
```

Comment:
As highlighted in the initial thread, would it be worth adding a simple logging line indicating the connection happened successfully?